src/app/core/basic-datatypes/entity/entity.datatype.spec.ts

```diff
@@ -23,50 +23,76 @@ import { EntitySchemaField } from "../../entity/schema/entity-schema-field";
 import { TestEntity } from "../../../utils/test-utils/TestEntity";
 
 describe("Schema data type: entity", () => {
-  testDatatype(new EntityDatatype(null, null), "1", "1", "User");
+  testDatatype(new EntityDatatype(null as any, null as any), "1", "1", "User");
 
-  it("should map to the referenced entity", async () => {
-    const c1 = TestEntity.create("first");
-    const c2 = new TestEntity();
-    c2.other = "123";
-    const entityMapper = mockEntityMapper([c1, c2]);
-    const dataType = new EntityDatatype(entityMapper, null);
-    const schema = TestEntity.schema.get("ref");
+  describe("importMapFunction", () => {
```

Comment:
these second level "describe(" blocks make it a bit hard to follow what actually changed in this ... For easier review, can you please undo these. And also undo the changes in entity.datatype.ts in the `anonymize()` method, as these are not related, I think?

Sorry for the inconvenience :blush:   but to keep our speed of code improvements and reviews high, this is an important aspect.